components/org.wso2.carbon.identity.scim.provider/src/main/java/org/wso2/carbon/identity/scim/provider/resources/UserResource.java

```diff
@@ -234,6 +234,12 @@ public Response updateUser(@PathParam(SCIMConstants.CommonSchemaConstants.ID) St
                                String resourceString) {
         Encoder encoder = null;
         try {
+            if (StringUtils.equals(id, "me")) {
+                SCIMCommonUtils.setThreadLocalToIdentifyMeEndpointCall(true);
```

Comment:
Shouldn't we unset the threadlocal, after the operation is finished similar to [1]

[1] - https://github.com/wso2-extensions/identity-inbound-provisioning-scim/blob/master/components/org.wso2.carbon.identity.scim.provider/src/main/java/org/wso2/carbon/identity/scim/provider/resources/UserResource.java#L369

Reply:
Fixed